packages/core/src/core.ts

```diff
@@ -102,6 +102,13 @@ export function setFailed(message: string): void {
 // Logging Commands
 //-----------------------------------------------------------------------
 
+/**
+ * Gets whether Actions Step Debug is on or not
+ */
+export function isDebug(): boolean {
+  return process.env['RUNNER_DEBUG'] === '1'
```

Comment:
@TingluoHuang I have set the `ACTIONS_RUNNER_DEBUG` secret, but the environment variable `RUNNER_DEBUG` does not seem to be set. Am I doing something wrong, or is this pending some update of the actions runner?

Reply:
It's in a runner that just started rolling out.

Reply:
@TingluoHuang looks like this may be ready to merge now. https://github.com/actions/runner/releases/tag/v2.165.2

Reply:
yes.